scheme/parsec-tpm/evidence_handler.go

```diff
@@ -226,7 +225,7 @@ func parsecTpmLookupKey(scope, tenantID, class, instance string) string {
 	return u.String()
 }
 
-func evidenceToMap(e parsectpm.Evidence) (map[string]interface{}, error) {
+func evidenceAsMap(e parsectpm.Evidence) (map[string]interface{}, error) {
```

Comment:
I meant:

```golang
func (e parsectpm.Evidence) AsMap() (map[string]interface{}, error) {
```

Reply:
Oh, ok I missed that let me check and correct it!

Reply:
No I checked, it is correct:
It is inline with the current ClaimsToMap, where individual claims were desired.

Here, we are expecting Evidence to return as a Map.
To move it to library, we can do it consistently using the other schemes then...

Reply:
This comment is no longer valid, hence resolving the same!